bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/i18n/avmfritz_de.properties

```diff
@@ -135,6 +135,10 @@ channel-type.avmfritz.comfort_temp.label = Komforttemperatur
 channel-type.avmfritz.comfort_temp.description = Zeigt die aktuell eingestellte Komforttemperatur des Heizkörperreglers an.
 channel-type.avmfritz.contact_state.label = Tür-/Fenster-Zustand
 channel-type.avmfritz.contact_state.description = Zeigt an, ob die Tür oder das Fester offen oder geschlossen ist (OPEN oder CLOSED).
+channel-type.avmfritz.obstruction_alarm.label = Hindernisalarm
+channel-type.avmfritz.obstruction_alarm.description = Hindernisalarm, der Rollladen wird gestoppt und ein kleines Stück in entgegengesetzte Richtung bewegt.
+channel-type.avmfritz.temperature_alarm.label = Temperaturalarm
+channel-type.avmfritz.temperature_alarm.description = Temperaturalarm des Rollladens, Motor überhitzt.
```

Comment:
Please do not update the German translations in that PR. They have to be changed later in Crowdin.

Reply:
Removed German translation